server/src/main/java/org/opensearch/index/mapper/BinaryRangeUtil.java

```diff
@@ -47,13 +47,12 @@
 import java.util.List;
 import java.util.Set;
 
-enum BinaryRangeUtil {
+final class BinaryRangeUtil {
```

Comment:
Yeah, I thought the same. I think it's because the ctor is private for enums? Weird choice over just making it a final class w/ an explicit private ctor. That's why I made the change.